.code-samples.meilisearch.yaml

```diff
@@ -3,10 +3,39 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
+getting_started_faceting: |-
+  $client->index('movies')->updateFaceting(['maxValuesPerFacet' => 2]);
+getting_started_pagination: |-
+  $client->index('movies')->updatePagination(['maxTotalHits' => 500]);
+synonyms_guide_1: |-
+  $client->index('movies')->updateSynonyms([
+    'great' => ['fantastic'],
+    'fantastic' => ['great'],
+  ]);
+date_guide_index_1: |-
+  $gamesJson = file_get_contents('games.json');
+  $games = json_decode($gamesJson);
+
+  $client->index('games')->addDocuments($games)
+date_guide_filterable_attributes_1: |-
+  $client->index('games')->updateFilterableAttributes(['release_timestamp']);
+date_guide_filter_1: |-
+  $client->index('games')->search('', [
+    'filter' => ['release_timestamp >= 1514761200 AND release_timestamp < 1672527600']
+  ])
```

Comment:
I agree that it is more legible this way, but in the others you modify like `faceted_search_filter_1` for example, you choose to make the return like this:
```
  $client->index('games')
    ->search('', ['filter' => ['release_timestamp >= 1514761200 AND release_timestamp < 1672527600']])
```
I personally don't have a preference but I think it is better if it is handled the same way